learn/advanced/asynchronous_operations.md

```diff
@@ -132,7 +132,7 @@ If the task was canceled while it was `enqueued` or `processing`:
   "details": {
     "matchedTasks": 9000,
     "canceledTasks": 0,
-    "originalQuery": "type=documentAdditionOrUpdate&documentDeletion"
+    "originalFilters": "types=documentAdditionOrUpdate&documentDeletion"
```

Comment:
@Kerollmops Are we displaying the `?` first char in `originalFilters`?

Could be helpful for copy-pasting.

Reply:
Hey @gmourier.
Just did a little test and it look like we don't do it for the cancelation or the deletion:
```json
  "details": {
    "deletedTasks": 1,
    "matchedTasks": 1,
    "originalQuery": "uid=0"
  },
```
And
```json
  "details": {
    "canceledTasks": 0,
    "matchedTasks": 1,
    "originalQuery": "uid=1"
  },
```

Reply:
Thanks, @irevoire! WDYT of including the `?` char?

Reply:
yep we could

Reply:
Let's go! Thanks, @irevoire, and @Kerollmops. I will update the specs accordingly. 

Signal to @maryamsulemani97

Reply:
Not sure I follow
Where does the `?` go?
`originalFilter": "?uid=1"`